docs/pages-for-subheaders/installation-requirements.md

```diff
@@ -11,41 +11,21 @@ If you install Rancher on a Kubernetes cluster, requirements are different from
 
 :::
 
-Make sure the node(s) for the Rancher server fulfill the following requirements:
-
-- [Operating Systems and Container Runtime Requirements](#operating-systems-and-container-runtime-requirements)
-  - [RKE Specific Requirements](#rke-specific-requirements)
-  - [K3s Specific Requirements](#k3s-specific-requirements)
-  - [RKE2 Specific Requirements](#rke2-specific-requirements)
-  - [Installing Docker](#installing-docker)
-- [Hardware Requirements](#hardware-requirements)
-- [CPU and Memory](#cpu-and-memory)
-  - [RKE and Hosted Kubernetes](#rke-and-hosted-kubernetes)
-  - [K3s Kubernetes](#k3s-kubernetes)
-  - [RKE2 Kubernetes](#rke2-kubernetes)
-  - [Docker](#docker)
-- [Ingress](#ingress)
-- [Disks](#disks)
-- [Networking Requirements](#networking-requirements)
-  - [Node IP Addresses](#node-ip-addresses)
-  - [Port Requirements](#port-requirements)
-- [Dockershim Support](#dockershim-support)
-
-For a list of best practices that we recommend for running the Rancher server in production, refer to the [best practices section.](../reference-guides/best-practices/rancher-server/tips-for-running-rancher.md)
-
-The Rancher UI works best in Firefox or Chromium based browsers (Chrome, Edge, Opera, Brave, ...).
+The Rancher UI works best in Firefox or Chromium based browsers (Chrome, Edge, Opera, Brave, etc).
 
-## Operating Systems and Container Runtime Requirements
+See our page on [best practices](../reference-guides/best-practices/rancher-server/tips-for-running-rancher.md) for a list of recommendations for running a Rancher server in production.
 
-Rancher should work with any modern Linux distribution.
+## Kubernetes Compatibility with Rancher
 
-Docker is required for nodes that will run RKE Kubernetes clusters. It is not required for RKE2 or K3s clusters.
+Rancher needs to be installed on a supported Kubernetes version. Consult the [Rancher support matrix](https://www.suse.com/suse-rancher/support-matrix) to ensure that your intended version of Kubernetes is supported.
 
-Rancher needs to be installed on a supported Kubernetes version. To find out which versions of Kubernetes are supported for your Rancher version, refer to the [support maintenance terms.](https://rancher.com/support-maintenance-terms/)
+## Operating Systems and Container Runtime Requirements
 
-For details on which OS and Docker versions were tested with each Rancher version, refer to the [support maintenance terms.](https://rancher.com/support-maintenance-terms/)
+All supported operating systems are 64-bit x86. Rancher should work with any modern Linux distribution.
 
-All supported operating systems are 64-bit x86.
+The [Rancher support matrix](https://www.suse.com/suse-rancher/support-matrix) lists which OS and Docker versions were tested for each Rancher version.
+
+Docker is required for nodes that will run RKE Kubernetes clusters. It is not required for RKE2 or K3s clusters.
```

Comment:
```suggestion
Docker is required for nodes that will run RKE clusters. It is not required for RKE2 or K3s clusters.
```

We should probably drop the `Kubernetes` here since it's a bit redundant.